### PR TITLE
fix: inconsistency and grammar in help messages

### PR DESCRIPTION
**Pull request by morinokami**
- Add periods for consistency where missing.
- Fix grammer.
- Update jsr to JSR.

### CODE REVIEW

File: src/bin.ts [hunk 1]
```diff
@@ -28,16 +28,16 @@ function printHelp() {
...
     ])
   }
 
 Commands:
 ${
     prettyPrintRow([
-      ["i, install, add", "Install one or more jsr packages"],
-      ["r, uninstall, remove", "Remove one or more jsr packages"],
+      ["i, install, add", "Install one or more jsr packages."],
+      ["r, uninstall, remove", "Remove one or more jsr packages."],
```
[the remark this answers is not in the document]
Comment (morinokami) [reply src/bin.ts:40]: Yeah, thanks for spotting this. I checked the docs and it looks like the official casing is JSR, so this should be corrected as well.

Screenshot 2024-02-29 at 20 59 47

File: src/bin.ts [hunk 2]
```diff
@@ -79,7 +79,7 @@ ${
 Environment variables:
 ${
     prettyPrintRow([
-      ["JSR_URL", "Use a different registry url for the publish command"],
+      ["JSR_URL", "Use a different registry url for the publish command."],
```
Comment (roj1512) [inline src/bin.ts:82]:
```suggestion
      ["JSR_URL", "Use a different registry for the publish command."],
```

File: src/bin.ts [hunk 3]
```diff
@@ -28,16 +28,16 @@ function printHelp() {
 Usage:
 ${
     prettyPrintRow([
-      ["jsr add @std/log", 'Install the "@std/log" package from jsr.io'],
-      ["jsr remove @std/log", 'Remove the "@std/log" package from the project'],
+      ["jsr add @std/log", 'Install the "@std/log" package from jsr.io.'],
+      ["jsr remove @std/log", 'Remove the "@std/log" package from the project.'],
```
Comment (roj1512) [inline src/bin.ts:32]: Personal suggestion, feel free to ignore.

```suggestion
      ["jsr add @std/log", 'Install the package @std/log.'],
      ["jsr remove @std/log", 'Remove the package @std/log from the project.'],
```

**Comment by morinokami** [comment]
@roj1512 Thanks for your suggestions! But I am not sure if these changes are appropriate, and I feel they are outside the scope of this PR, so I will leave it to the maintainers to decide if they should be merged.

**Comment by bartlomieju** [comment]
Thanks for the PR, please sign the CLA so we can merge it :)

**Comment by morinokami** [comment]
@bartlomieju Signed it! 😁

**Review by marvinhagemeister — APPROVED** [review]
Sweet, thanks for ensuring consistent grammar 👍

**Pull request merged**